CMakeLists.txt

```diff
@@ -117,9 +117,11 @@ if(eoserv_GCC OR eoserv_CLANG)
 		if(eoserv_CXX1Y)
```

Comment:
You're dropping C++14 support, so `eoserv_CXX1Y` is no longer relevant and can be removed.